frontend/src/components/common/TabNavigation.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Tab from "./Tab";
+
+class TabNavigation extends Component {
+  static propTypes = {
+    tabSpecs: PropTypes.array.isRequired,
+    curTab: PropTypes.number.isRequired
+  };
+
+  state = {
+    tabSpecs: this.props.tabSpecs,
+    curTab: this.props.curTab,
+    curBody: this.props.tabSpecs[this.props.curTab].body
+  };
+
+  selectTab(id) {
+    let tempSpecs = this.state.tabSpecs;
+    tempSpecs[this.state.curTab].selected = "false";
```

Comment:
No need for quotes around booleans.

Reply:
I was using a string because it was not allowing a boolean as a prop (for Tab)